ads/vendors/andbeyond.js

```diff
@@ -0,0 +1,35 @@
+import {loadScript, validateData} from '#3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function andbeyond(global, data) {
+  validateData(
+    data,
+    ['siteid', 'slotpath', 'width', 'height'],
+    ['totalampslots', 'jsontargeting', 'extras']
+  );
+  loadScript(
+    global,
+    'https://securepubads.g.doubleclick.net/tag/js/gpt.js',
+    () => {
+      loadScript(
+        global,
+      'https://rtbcdn.andbeyond.media/prod-global-' + data.siteid + '.js',
```

Comment:
Nit: this is why it failed. The indentation needs to be fixed by adding two spaces.

Reply:
Fixed